src/pages/chat/components/ExchangeSidePreview.tsx

```diff
@@ -520,7 +522,7 @@ export default function ExchangeSidePreview({
       <Section>
         <DetailTable align noBorder data={OFFER_DETAIL_DATA ?? ({} as never)} />
       </Section>
-      {isInDispute && iAmTheBuyer && !isEscalated && !isRetracted ? (
+      {isInDispute && iAmTheBuyer && !isRetracted ? (
         <CTASection>
           <Button
```

Comment:
Retract should not be available in case Dispute was escalated and escalation period has elapsed